javascript/javascript1/week1/exercise4.js

```diff
@@ -0,0 +1,37 @@
+// Exercise 4 Ez Namey (Startup name generator)
+
+const randomNumber = Math.floor(Math.random() * 10);
+const firstWords = [
+	"Easy",
+	"Awesome",
+	"Corporate",
+	"Modern",
+	"Cool",
+	"Cute",
+	"Cozy",
+	"Crazy",
+	"Big",
+	"New",
+];
+const secondWords = [
+	"Way",
+	"Bussiness",
+	"Startup",
+	"Deal",
+	"Future",
+	"Life",
+	"Day",
+	"Adventure",
+	"Fairytale",
+	"Prank",
+];
+
+const startupName = firstWords[randomNumber] + " " + secondWords[randomNumber];
```

Comment:
The exercise is to match the two words at ramdom. By only having one random index accessor we always get the words in parallel. Make two random numbers, and use only one per array to match at random